src/components/stepforms/ReplaceTextStepForm.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <StepFormHeader
+      :title="title"
+      :stepName="editedStep.name"
+      :version="version"
+      :backendError="backendError"
+    />
+    <ColumnPicker
+      class="column"
+      v-model="editedStep.searchColumn"
+      name="Extract a substring from..."
+      placeholder="Enter a column"
+      data-path=".column"
+      :errors="errors"
+    />
+    <InputTextWidget
+      v-model="editedStep.oldStr"
+      name="Old string"
+      placeholder="Text to replace"
```

Comment:
```suggestion
      placeholder="String to replace"
```

Reply:
I figured "text" would be easier to understand by non-technical users

Reply:
Also, it's the name of the step :slightly_smiling_face: